rest/taskana-rest-spring/src/main/java/pro/taskana/rest/WorkbasketAccessItemController.java

```diff
@@ -86,6 +100,36 @@
         return new ResponseEntity<>(pagedResources, HttpStatus.OK);
     }
 
+    /**
+     * This DELETE method delete all workbasketAccessItems that correspond the given accessId.
+     *
+     * @param accessId which need remove his workbasketAccessItems.
+     * @return ResponseEntity if the user is not authorized.
+     * @throws NotAuthorizedException if the user is not authorized.
+     */
+    @DeleteMapping("{accessId}")
+    public ResponseEntity<Void> removeWorkbasketAccessItems(@PathVariable("accessId") String accessId)
```

Comment:
And one more comment to consistency: the path variable should be a uinque id. "accessId" is not identifying a single WorkbasketAccessItem. It can identify a list of items. Therefore we should not use accessId as a PathVariable but rather a Parameter. The call would than look like:
"DELETE /v1/workbasket-access-items?access-id=%%USERID%%"

Reply:
Done!